css-fonts/styles.css

```diff
@@ -0,0 +1,8 @@
+@import url('https://fonts.googleapis.com/css2?family=Merriweather:ital@1&family=Poppins:wght@100&display=swap');
```

Comment:
This is another way to import fonts into your page! However, you don't need to use the `@import` rule here if you are using the `<link>` tag in your HTML file (they are doing the same thing).